canonical/ubuntu.com
# WD-11736 build /pro/devices page

## Done

- Build /pro/devices page and its contact form 

## QA

- Go to /pro/devices
- Compare page against [copy doc](https://docs.google.com/document/d/1i_3nPkSpCQJ4jfOwN380sdfuTxaiDuDIgDfk01wVPqo/edit)
- Check CTA button for the form

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-11736
